jsk_aibo_robot/aibo_selenium_ros/python/aibo_selenium_ros/__init__.py

```diff
@@ -0,0 +1,184 @@
+import base64
+import io
+import logging
+import random
+import time
+
+import selenium
+from imageio import imread
```

Comment:
add iamgeio to pcakge.xml or requirements.txt

Reply:
Added